ComposureWatch/src/Components/PlayerCards.jsx

```diff
@@ -1,6 +1,160 @@
 const PlayerCards = (props) => {
-  if (props.name == "moira") {
-    return (cardImg = <img src="src/IMG/portrait-moira.png" />);
+  const charList = [
```

Comment:
Just an idea...you could make an object where the key is an "identifier" for each character. A couple benefits:

- Separate than the name
- Can be used for your `character="winston"` property
- Will disallow duplicates, i.e., you cannot have two "winston"

For example:

```
const charList = {
  winston: {
    name: "Winston",
    nickname: "Monkey",
    img: // etc...
  },
  echo: {
    name: "Echo",
    nickname: "Gnat"
  },
}
```

Then getting the data for this you could do something like `charList[props.character].name` 👍